app/controllers/organizations_controller.rb

```diff
@@ -101,7 +101,14 @@ def remove_user
     redirect_to settings_invitations_organization_path
   end
 
-  def new_assignment; end
+  # rubocop:disable Style/GuardClause
+  def new_assignment
+    if current_organization.archived?
```

Comment:
Would it make sense to use the guard clause here rather than disabling the rubocop linter for it?